packages/server/database/migrations/20210624120419-addPokerStats.ts

```diff
@@ -8,10 +8,12 @@ export const up = async function(r: R) {
         .filter({phaseType: NewMeetingPhaseTypeEnum.ESTIMATE})('stages')
         .concatMap((row) => row('discussionId'))
     }
-    const getStories = (meetingRow) => {
+    const getCompletedStories = (meetingRow) => {
       return meetingRow('phases')
         .filter({phaseType: NewMeetingPhaseTypeEnum.ESTIMATE})('stages')
-        .concatMap((row) => row('serviceTaskId'))
+        .nth(0)
+        .filter({isComplete: true})
```

Comment:
@nickoferrall it looks like this is filtering the 0th element. do we want to run the filter 1 line above?

Reply:
![nested-stages](https://user-images.githubusercontent.com/39854876/123482023-34bf6280-d5ca-11eb-9a38-575678678eb7.png)

`stages` is a nested array so I'm using `nth(0)` to access it. If I try to filter on `stages` without `nth(0)`, I get an error as `isComplete` isn't accessible. I tested the migration locally to make sure the stats are accurate in the timeline and it seems to be doing the business

Reply:
Ohhhh! i see what it's doing.
the filter is returning an array of estimate phases & you're grabbing the stages from each one & then limiting it to the first phase. good deal :+1: